odoo/http.py

```diff
@@ -1270,6 +1270,9 @@ def __getitem__(self, item):
         if item == 'longitude':
             return self.location.longitude
 
+        if item == 'accuracy_radius':
+            return self.location.accuracy_radius
+
```

Comment:
```suggestion
```
>     # Old dict API, undocumented for now, will be deprecated some day

Previously `request.geoip` was a dictionary with only those key insides. When we created this class, we decided to expose all the attributes, but we left this dict-like API for backward compatibility. There was a lot of `request.geoip.get('country_name')` and we didn't want to bloat the PR with changes like `request.geoip and request.geoip.country_name`.

Let's not add new entries.